org.starchartlabs.eclipse.template.dynamic.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.starchartlabs.eclipse.template.dynamic.feature"
       label="Java Template Dynamic Variables"
-      version="0.1.0.0"
```

Comment:
I would like to avoid editing the version numbers or binary jars in a pull request like this

Reply:
That's perfectly fine, just git reset it to the original for the jars/version number files and that should take care of it